app/Domain/Goalcanvas/Templates/dashboard.blade.php

```diff
@@ -0,0 +1,322 @@
+                            <li><a href="#/goalcanvas/bigRock">{{ __("links.icon.create_new_board") }}</a></li>
+                        @endif
+                        <li class="border"></li>
+                        @foreach ($tpl->get('allCanvas') as $canvasRow)
+                            <li><a href="{{ BASE_URL }}/{{ $canvasName }}canvas/showCanvas/{{ $canvasRow['id'] }}">{{ $tpl->escape($canvasRow['title']) }}</a></li>
+                        @endforeach
+                    </ul>
+                </span>
+            @endif
+        </h1>
+    </div>
+</div><!--pageheader-->
+
+<div class="maincontent">
+
+{!! $tpl->displayNotification() !!}
+
+<div class="row" style="margin-bottom:20px; ">
+    <div class="col-md-4">
+        <div class="bigNumberBox" style="padding: 29px 15px;">
+            <h2>Progress: {{ round($goalStats['avgPercentComplete']) }}%</h2>
+
+            <div class="progress" style="margin-top:5px;">
+                <div class="progress-bar progress-bar-success" role="progressbar"
+                     aria-valuenow="{{ round($goalStats['avgPercentComplete']) }}" aria-valuemin="0" aria-valuemax="100"
+                     style="width: {{ $goalStats['avgPercentComplete'] }}%">
+                    <span class="sr-only">{{ sprintf(__("text.percent_complete"), round($goalStats['avgPercentComplete'])) }}</span>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="col-md-2"></div>
+    <div class="col-md-2">
+        <div class="bigNumberBox priority-border-4">
+            <h2>On Track</h2>
+            <span class="content">{{ $goalStats['goalsOnTrack'] }}</span>
+        </div>
+    </div>
+    <div class="col-md-2">
+        <div class="bigNumberBox priority-border-3">
+            <h2>At Risk</h2>
+            <span class="content">{{ $goalStats['goalsAtRisk'] }}</span>
+        </div>
+    </div>
+    <div class="col-md-2">
+        <div class="bigNumberBox priority-border-1">
+            <h2>Miss</h2>
+            <span class="content">{{ $goalStats['goalsMiss'] }}</span>
+        </div>
+    </div>
+</div>
+
+<div class="maincontentinner">
+    <div class="row">
+        <div class="col-md-6"></div>
+    </div>
+    @if (count($allCanvas) > 0)
+        @foreach ($tpl->get('allCanvas') as $canvasRow)
+            <div class="row">
+                <div class="col-md-12">
+                    <a href='#/goalcanvas/editCanvasItem?type=goal&canvasId={{ $canvasRow["id"] }}' class='btn btn-primary pull-right'><i class="fa fa-plus"></i> Create New Goal</a>
+
+                    <h5 class='subtitle'><a href='{{ BASE_URL }}/{{ $canvasName }}canvas/showCanvas/{{ $canvasRow["id"] }}'>{{ $tpl->escape($canvasRow["title"]) }}</a></h5>
+                </div>
+            </div>
+            <div class="row" style="border-bottom:1px solid var(--main-border-color); margin-bottom:20px">
+                @php
+                $canvasSvc = app()->make(Goalcanvas::class);
```

Comment:
FYI, no change required right now: If you are working on the service layer we should probably just build the entire array in there and not make those calls in the template.